models/SequenceVerificationDataCentric/models/SequenceVerificationDataCentric/SVDCdomain/AESequenceDC/SVDCclasses/JobDefinition/JobDefinition.masl

```diff
@@ -157,8 +157,10 @@ begin
     delete theConstraintDefinitions;
     delete this;
     
-    logMessage := "AESequenceDC::JobDefinition.deleteJobDefinition : Job Definition " & jobName & " has been deleted";
-	Logger::log(Logger::Debug, "AESequenceDC", logMessage);
+    if (Logger::enabled(Logger::Debug, "pv.sv.jobdefinition.deletejobdefinition")) then
+      logMessage := "AESequenceDC::JobDefinition.deleteJobDefinition : Job Definition " & jobName & " has been deleted";
+      Logger::log(Logger::Debug, "pv.sv.jobdefinition.deletejobdefinition", logMessage);
```

Comment:
Should this be pv.sv.jobdefinition.deleted (as for .../JobDefinition/InstanceStateMachine/InstanceStateMachine.masl above)?

Reply:
I made this one match the operation.  The operation could be called from other states.